docs/api/users/create.md

```diff
@@ -1,7 +1,136 @@
+
+  <Tabs
+    defaultValue="nodejs"
+    values={[
+      { label: 'NodeJS', value: 'nodejs' },
+      { label: 'Bash', value: 'bash' }
+    ]
+  }>
+  <TabItem value="nodejs">
+
+  Copy the below code at the bottom of your nodejs script that handles the authentication of the OpenHIM headers as described in the [authentication section](../introduction/authentication.md). 
+
+  Replace the `openhimOptions` values with the correct implementation details and supply the `SampleData` payload to submit
+
+  ```javascript
+  // append below code to the "openhim-api.js" script containing the authentication methods. 
+  // This is described within the authentication section
+
+  (async () => {
+    const openhimOptions = {
+      apiURL: 'https://localhost:8080',
+      apiEndpoint: '/users',
+      username: 'root@openhim.org',
+      password: 'openhim-password',
+      rejectUnauthorized: false
+    }
+    const SampleData = 'SampleData'
+
+    const headers = await genAuthHeaders(openhimOptions)
+
+    const options = { 
+      method: 'POST',
+      url: `${openhimOptions.apiURL}${openhimOptions.apiEndpoint}`,
+      rejectUnauthorized: openhimOptions.rejectUnauthorized,
+      headers: headers,
+      body: SampleData,
+      json: true
+    }
+
+    request(options, (error, response, body) => {
+      if (error) throw new Error(error)
+
+      console.log({
+        statusCode: response.statusCode,
+        body
+      })
+    })
+  })()
+  ```
+
+   Execute the below command in your terminal to run the nodejs script
+
+  ```bash
+  node openhim-api.js
+  ```
+
+  </TabItem>
+  <TabItem value="bash">
+
+  Ensure that you have created your bash script to construct the HTTP authentication headers and send the request to the OpenHIM API as described in the [authentication section](../introduction/authentication.md). 
+
+  Execute the below command in your terminal where the file is located with the required arguments. Replace the placeholder arguments with the correct implementation details and ensure your `SampleData.json` file exists with the user object.
+
+  ```curl
+  ./openhim-api.sh root@openhim.org openhim-password -v https://localhost:8080/users -d @SampleData.json -H "Content-Type:application/json"
+  ```
+
+  </TabItem>
+  </Tabs>
+
+  The response status will be 201 if successful.
+  
+  Once created, a user's activation email is sent to the users email. For this email to be sent, the OpenHIM's email and password have to be set up.
+
+</TabItem>
+<TabItem value="sample">
+
+  ```json
+  {
+    "settings": {
+      "list": {},
+      "filter": {
+        "limit":100
+      }
+    },
+    "groups": ["root@openhim.org"],
+    "email": "root2@openhim.org",
+    "firstname": "Test",
+    "surname": "Tester",
+    "msisdn": "",
+    "passwordAlgorithm": "sha512",
+    "passwordSalt": "32d0efc6add6dbd29ed46060a2744e0f",
```

Comment:
We might need to include some additional example logic to help the user build the correct salt/hash. If you have a look at the changes i made in the `Clients` section you can re-use that